fe/be-java-extensions/java-udf/src/main/java/org/apache/doris/udf/BaseExecutor.java

```diff
@@ -274,6 +274,28 @@ protected ColumnValueConverter getInputConverter(TPrimitiveType primitiveType, C
                     return result;
                 };
             }
+            case VARBINARY: {
+                // Support Byte[] parameter for VARBINARY by boxing from byte[]
```

Comment:
check most common use `Byte` or `byte`

Reply:
Hive use BytesWritable, and it's could  regard as byte[]